### PR TITLE
DHIS2 6276 booleans

### PR DESCRIPTION
**Pull request by tomzemp**
This PR updates maintenance app to display booleans: https://jira.dhis2.org/browse/DHIS2-6276

The code will also stringify data types other than null, but these should not be returned from a request to sqlViews resource.

Null was mapped to an empty string to make explicit the choice of no display for this data type.

### CODE REVIEW

File: src/SqlView/SqlView.component.js [hunk 1]
```diff
@@ -138,8 +138,9 @@ class SqlView extends Component {
 
     /* eslint-disable react/no-array-index-key */
     renderCell = (cell, index) => {
-        const isObject = typeof cell === 'object' && cell != null;
-        const formattedValue = isObject ? JSON.stringify(cell, null, 2) : cell;
+        const formattedValue =
+            cell === null ? '' :
+            (typeof cell === 'object' ? JSON.stringify(cell, null, 2) : String(cell));
```
Comment (Mohammer5) [inline src/SqlView/SqlView.component.js:143]: This is quite hard to parse. I think this should be outsourced into its own function for better readability:

```js
const formatCellValue = (cell) => {
  if (cell === null) {
    return null
  }

  if (typeof cell === 'object') {
    return JSON.stringify(cell, null, 2)
  }

  return String(cell)
}
```

**Review by Mohammer5 — CHANGES_REQUESTED** [review]

File: src/SqlView/SqlView.component.js (hunk 1 above)
Comment (tomzemp) [reply src/SqlView/SqlView.component.js]: Logic moved to function

Comment (tomzemp) [reply src/SqlView/SqlView.component.js]: Hi @Mohammer5 : I've moved to logic into a function. If you'd like any other changes, let me know

**Review by Mohammer5 — APPROVED** [review]

**Comment by Mohammer5** [comment]
Sorry for the late response.. Somehow this went under the radar..
